awx/main/tasks/host_indirect.py

```diff
@@ -24,5 +31,31 @@ def build_indirect_host_data(job, event_query: dict):
                 continue
             canonical_facts = data['canonical_facts']
             facts = data.get('facts')
-            results.append(SimpleNamespace(canonical_facts=canonical_facts, facts=facts))
+            results.append(IndirectManagedNodeAudit(canonical_facts=canonical_facts, facts=facts, job=job, organization=job.organization))
```

Comment:
do we need to set the other fields (inventory, host, events etc.) here too?

Reply:
Oh I neglected the inventory field. I'll have to read-read about what the events field is...